packages/cli/src/api/catalog.ts

```diff
@@ -251,8 +251,9 @@ export class Catalog {
     this.locales.forEach((locale) => this.write(locale, catalogs[locale]))
   }
 
-  writeCompiled(locale: string, compiledCatalog: string) {
-    const filename = this.path.replace(LOCALE, locale) + ".js"
+  writeCompiled(locale: string, compiledCatalog: string, namespace?: string) {
+    const ext = `.${ namespace === "es" ? "m": ""}js`
```

Comment:
Could you please rewrite this condition so it's easier to find it? E.g. when someone search for `mjs` across the codebase

```suggestion
    const ext = `.${ namespace === "es" ? "mjs": "js"}`
```

Reply:
Good idea. I am not super familiar with the Github UI. It offered me an option to `Commit suggestion`, so just did it. Hope it will not mess up the history.